mdns.h

```diff
@@ -22,8 +22,8 @@
 
 #include <fcntl.h>
 #ifdef _WIN32
-#include <Winsock2.h>
```

Comment:
There is already a PR in review for the header fix: https://github.com/analogdevicesinc/libiio/pull/567